zio-http/src/test/scala/zio/http/ContentTypeSpec.scala

```diff
@@ -42,11 +42,11 @@ object ContentTypeSpec extends HttpRunnableSpec {
   )
 
   private val env =
-    EventLoopGroup.nio() ++ ChannelFactory.nio ++ ServerChannelFactory.nio ++ DynamicServer.live ++ Scope.default
+    DynamicServer.live ++ Server.test ++ ChannelFactory.nio ++ EventLoopGroup.nio(0)
```

Comment:
Don't wire up manually, instead, use the `.provide()` macro below.

Reply:
Done